src/sentry/models/organizationmember.py

```diff
@@ -235,6 +235,54 @@ def send_invite_email(self):
             logger = get_logger(name="sentry.mail")
             logger.exception(e)
 
+    def send_request_notification_email(self):
+        from sentry.utils.email import MessageBuilder
+
+        link_args = {"organization_slug": self.organization.slug}
+
+        context = {
+            "email": self.email,
+            "inviter": self.inviter,
+            "organization": self.organization,
+            "organization_link": absolute_uri(
+                reverse("sentry-organization-index", args=[self.organization.slug])
+            ),
+            "pending_requests_link": absolute_uri(
+                reverse("sentry-organization-members-requests", kwargs=link_args)
+            ),
+        }
+
+        if self.requested_to_join:
+            email_args = {
+                "subject": "%s has requested to join %s on Sentry"
+                % (self.email, self.organization.name),
+                "template": "sentry/emails/organization-join-request.txt",
+                "html_template": "sentry/emails/organization-join-request.html",
+            }
+        elif self.requested_to_be_invited:
+            email_args = {
+                "subject": "%s has requested to invite a member to %s on Sentry"
```

Comment:
maybe these subject lines could be shorter and less specific like `Sentry Access Request`

Reply:
Ahh yeah, good call